src/Enums.js

```diff
@@ -32,12 +32,31 @@ export const membershipStatus = [
   { key: 'Admin', value: 2 }
 ]
 
+export const officerRole = {
+  DEVELOPMENT: 'Development',
+  PUBLICRELATION: 'Public Relation',
+  EVENTPLANNING: 'Event Planning',
+  PRINTING: '3D-Printing'
+}
+
+export const chairRole = {
+  PRESIDENT: 'President',
+  VICEPRESIDENT: 'Vice President',
+  TREASURER: 'Treasurer',
+  DEVELOPMENTCHAIR: 'Software Development Chair',
+  EVENTPLANNINGCHAIR: 'Event Planning Chair',
+  STOREMANAGER: 'Store Manager',
+  PRINTINGCHAIR: '3D-Printing/Hacking Station Chair'
```

Comment:
```suggestion
  PRINTING_SPECIALIST: '3D-Printing/Hacking Station Chair'
```
name it like this:
![image](https://user-images.githubusercontent.com/36345325/72775915-cd7fd080-3bd5-11ea-8c14-8d69d2a583d9.png)